modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /etc/kms-secrets
+                    name: kms-secrets
+                volumes:
+                -  name: kms-secrets
+                   secret:
+                    secretName: kms-secrets
+# ...
+----
+
+* With this update, you can configure a list of namespaces as an argument to the {tekton-chains} controller. If you provide this list, {tekton-chains} watches pipeline runs and task runs only in the specified namespaces. If you do not provide this list, {tekton-chains} watches pipeline runs and task runs in all namespaces.
++
+.Example configuration for watching only the `dev` and `test` namespaces
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - args:
+                  - --namespace=dev, test 
+                  name: tekton-chains-controller
+----
+
+[id="tekton-results-new-features-1-16_{context}"]
+=== {tekton-results}
+
+* Before this update, {tekton-results} used the `v1beta1` API format to store `TaskRun` and `PipelineRun` object records. With this update, {tekton-results} uses the `v1` API format to store `TaskRun` and `PipelineRun` object records.
+
+* With this update, {tekton-results} can automatically convert existing records to the `v1` API format. To enable such conversion, set parameters in the `TektonResult` CR as shown in the following example:
++
+.Example of configuring {tekton-results} to convert existing records to the `v1` API format
+[source,yaml]
+----
+  apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonResult
+metadata:
+  name: result
+spec:
+  options:
+    deployments:
+      tekton-results-api:
+        spec:
+          template:
+            spec:
+              containers:
+                - name: api
+                  env:
+                    - name: CONVERTER_ENABLE
+                      value: "true"
+                    - name: CONVERTER_DB_LIMIT
+                      value: "256" #<1>
+----
+<1> In the `CONVERTER_DB_LIMIT` variable, set the number of records to convert at the same time in a single transaction.
+
+* With this update, {tekton-results} now supports fetching forwarded logs from third party logging APIs. You can enable the logging API through the `TektonResult` CR by setting the `logs_api` to `true` and `logs_type` to `Loki`.
+
+* With this update, you can configure automatic pruning of the {tekton-results} database. You can specify the number of days for which records must be stored, as well as the schedule for running the pruner job that removes older records. To set these parameters, edit the `TektonResult` CR, as shown in the following example:
```

Comment:
```suggestion
* With this update, you can configure automatic pruning of the {tekton-results} database. You can specify the number of days for which records must be stored, and the schedule for running the pruner job that removes older records. To set these parameters, edit the `TektonResult` CR, as shown in the following example:
```

Reply:
I don't think we can accept this change as proposed. ", and", with a comma, breaks the grammar. Without a comma, "the number of days for which records must be stored and the schedule for running the pruner job", it seems rather hard to parse.

However there is a way to handle this. "With this update, you can configure automatic pruning of the {tekton-results} database. You can specify the number of days for which records must be stored. You can also specify the schedule..."